src/static/themes/dark/dark.css

```diff
@@ -232,6 +232,8 @@
     --tridactyl-fg: var(--tridactyl-photon-colours-in-content-page-color);
     --tridactyl-bg: var(--tridactyl-photon-colours-in-content-page-background);
 
+    --tridactyl-scrollbar-color: var(--tridactyl-bg) var(--tridactyl-fg);
```

Comment:
In theory I like this solution, but it doesn't seem to do anything on my machine - the scroll bar appears to be unchanged from the default.

Reply:
Tue Oct 13 16:48:21 BST 2020

Are you testing on the `help` page?

#### dark: bg: pink

<img width="1680" alt="tridactyl-help-dark-bg=pink" src="https://user-images.githubusercontent.com/278414/95884333-7de8c280-0d6b-11eb-9a75-9fabd3b918f8.png">

#### dark: bg: darkgreen

<img width="1680" alt="tridactyl-help-dark-bg=darkgreen" src="https://user-images.githubusercontent.com/278414/95884353-8214e000-0d6b-11eb-8990-5da67dd7cb4d.png">

The above were set using the developer tools to change the colours.  The scrollbar changes to match.

My workflow - after making changes to `css` files:
```sh
: ; rm -rf generated build && yarn run build && yarn run run
```
##### Note:
So far, the `scrollbar` changes only affect the `help` page.

Reply:
> My workflow - after making changes to css files

Aha, that was my problem, somehow only some of the CSS was getting updated. It works for me now

I'll have a final look tomorrow and then, if all goes well, merge it.